python/tank/authentication/errors.py

```diff
@@ -41,6 +41,21 @@ def __init__(self, msg):
         )
 
 
+class ConsoleLoginWithSSONotSupportedError(ShotgunAuthenticationError):
+    """
+    Thrown when attempting to use Username/Password pair to login onto
+    a SSO-enabled site.
+    """
+
+    def __init__(self, url):
+        """
+        :param str url: Url of the site where login was attempted.
+        """
+        ShotgunAuthenticationError.__init__(
+            self, "Authentication using username/password is not allowed on the console for %s, an SSO-enabled site." % url
```

Comment:
nitpick: "is allowed" seems like the wrong wording? How about "is supported"?

Reply:
I've re-used the wording used in Shotgun. If I change it in one place, I need to update the other.